pkg/util/util.go

```diff
@@ -298,3 +301,46 @@ func MustNewRequirement(key string, op selection.Operator, vals []string) *label
 	}
 	return r
 }
+
+func AppendAdditionalVolumeAndVolumeMount(volMounts []corev1.VolumeMount, storageVolumes []v1alpha1.StorageVolume, tc *v1alpha1.TidbCluster, memberType v1alpha1.MemberType) ([]corev1.PersistentVolumeClaim, []corev1.VolumeMount) {
```

Comment:
What about we change the function signature as below and then we just append them in the calling functions:
```
func BuildAdditionalVolumeAndVolumeMount(tc *v1alpha1.TidbCluster, memberType v1alpha1.MemberType) ([]corev1.PersistentVolumeClaim, []corev1.VolumeMount) {
```